### ISSUE TITLE
feat: View default namespace for authorised users

### ISSUE THREAD

**Issue by portly-halicore-76** [belastingdienst/opr-paas#162]
# Feature Request

**Describe the Feature Request**

Give Paas users, defined via the paas.spec.groups, view access to the default namespace (named after the paas itself) generated by the operator. 

Why:

PaasNs'es belonging to the Paas capabilities, are created in this default namespace. If, for some reason, an error is thrown in the reconciliation of these PaasNs'es, is it stated on the status block of those PaasNs'es in that default namespace. However, only a clusteradmin has rights to that namespace, as Paas users from the Paas, don't get access. So far self-service for our customers. 

**Describe Preferred Solution**

Reconcile a view rolebinding for the default namespace, to each group specified in the Paas.
Document that a permission must be in place that all authenticated users may view PaasNs'es.

**Describe Alternatives**

Can't think of any.

**Related Code**

Probably in the rolebinding controller.

**Additional Context**

**If the feature request is approved, would you be willing to submit a PR?**
_(Help can be provided if you need assistance submitting a PR)_

- [x] Yes
- [ ] No

**Comment by devotional-phoenix-97** [comment]
We previously decided not to grant any permissions, since removing the PaasNs would remove the namespace with the app.
Of course we can grant only a view role on the paasns's in this namespace.

**Comment by portly-halicore-76** [comment]
> We previously decided not to grant any permissions, since removing the PaasNs would remove the namespace with the app. Of course we can grant only a view role on the paasns's in this namespace.

Exactly. So the scope of this issue is only to add the view right on the default namespace. Dealing with the RBAC related to deleting  / viewing of PaasNs'es in that namespace is out of scope.

**Comment by portly-halicore-76** [comment]
Let's discuss beginning of januari as there are multiple ways to solve this issue.

**Comment by portly-halicore-76** [comment]
No longer relevant as closed by #598
